src/github.com/getlantern/flashlight/proxiedsites/proxiedsites.go

```diff
@@ -52,6 +56,24 @@ func Configure(cfg *proxiedsites.Config, proxyAddr string) {
 	startMutex.Unlock()
 }
 
+func updateDetour(delta *proxiedsites.Delta) {
+	// TODO: subscribe changes of geolookup and set country accordingly
+	// safe to hardcode here as IR has all detection rules
+	detour.SetCountry("IR")
```

Comment:
The detection rule itself is rather simple and it's easy to change detect.go.  Addition of rules may requires change of detour's logic. I think it just works now.